deploy-all-in-one-tls.yaml

```diff
@@ -257,6 +257,14 @@ stringData:
         channel: 'MATTERMOST_CHANNEL'               # Mattermost Channel for receiving BotKube alerts
         notiftype: short                            # Change notification type short/long you want to receive. notiftype is optional and Default notification type is short (if not specified)
 
+      # Settings for Discord
+      discord:
+        enabled: false
+        token: 'DISCORD_TOKEN'	        # BotKube Bot Token 
```

Comment:
Fix indentation here and similar other occurances

Reply:
done